boa/src/builtins/console.rs

```diff
@@ -32,11 +32,34 @@ pub fn error(_: &Value, args: &[Value], _: &mut Interpreter) -> ResultValue {
     Ok(Gc::new(ValueData::Undefined))
 }
 
+/// Print a javascript value to the standard error only if first argument evaluates to false or
+/// there were no arguments
```

Comment:
I would do as Rust suggest, to include a small explanation of one line at the beginning and then maybe some extra information in the 3rd line of comment, so that it shows nicer when generating the documentation.

I would add the link to the spec: https://console.spec.whatwg.org/#assert

Reply:
Ok, I'll do it.